### DISCUSSION TITLE
Compat mode with Blade components?

### DISCUSSION THREAD

**Discussion by jordyvandomselaar** [alpinejs/alpine#860] [1. Help]
Blade components use `:` as well like so:

```<x-component :value="$value" />```

Is there a way not to use `:attr` or `x-bind:attr`? Laravel also doesn't like `x-bind:attr` as it also contains a `:`.

**Comment by ryangjchandler** [comment]
Using `x-bind:value` should be fine, since Blade components are also compatible with Livewire which uses `:` inside of attributes such as `wire:model`.

**Reply by jordyvandomselaar** [reply]
Aah, seems your right! The errors were PEBCAK, thanks for the reply =)
